packages/devops/scripts/lambda/actions/spin_up_non_tupaia_deployment.py

```diff
@@ -0,0 +1,37 @@
+# Creates a new deployment of DHIS2 or another EC2 instance hosted within the tupaia.org domain, but
+# not Tupaia proper (which has a separate file, see spin_up_tupaia_deployment)
+#
+# Example configs
+#
+# 1. Spin up new deployment of Tonga DHIS2
+# {
+#   "Action": "spin_up_non_tupaia_deployment",
+#   "User": "edwin",
+#   "DeploymentName": "tonga-for-testing",
+#   "InstanceType": "t3a.medium",
+#   "FromDeployment": "tonga"
+# }
+
+from helpers.clone import clone_instance
+
+def spin_up_non_tupaia_deployment(event):
```

Comment:
Yeah that's more straightforward huh. This can be used for other deployments that have a backup process in place, but they do need all the right tags etc, and atm it is only the DHIS2 ones that have those set up, so will rename.